### DISCUSSION TITLE
Create second server from docker Synology

### DISCUSSION THREAD

**Discussion by MAL0W** [lloesche/valheim-server-docker#174] [Q&A]
Please, what would be the variables to create a second server from Synology Docker?

**Comment by lloesche** [comment]
I don't think you would have to change any variables at all. Just mount a fresh volume to `/config` and `/opt/valheim` and map container ports `2456-2458` to something else on your Synology.

That is in this screen:
![Step 5](https://raw.githubusercontent.com/lloesche/valheim-server-docker/main/misc/step5.png "Step 5")

You use different "Local Ports" but the "Container Ports" stay the same.
For instance map 2466 to 2456, 2467 to 2457 and 2468 to 2458. And then in Steam/Valheim you connect to your Synology IP:2466.

**Reply by MAL0W** [reply]
First, I want to thank you for your support.

Would it be like this?

SERVER 1
![server 1 ports](https://user-images.githubusercontent.com/5898238/110381662-1274f980-805a-11eb-8d44-baa5e8ce44ce.png)
![server 1 volumen](https://user-images.githubusercontent.com/5898238/110381674-19037100-805a-11eb-94d9-3bbb038c8d6d.png)

SERVER2
![server 2 ports](https://user-images.githubusercontent.com/5898238/110381710-24569c80-805a-11eb-8b3c-8c5e57e8f64d.png)
![server 2 volumen](https://user-images.githubusercontent.com/5898238/110381719-26b8f680-805a-11eb-8b8d-042142ad8086.png)


The montage you tell me about `/opt/valheim` I don't know to which folder path I would have to put it

**Reply by lloesche** [reply]
This looks correct! `/opt/valheim` is where the container downloads and installs the server to. Without it every time you download a new version of the Docker image the container will have to re-download 1 GB of Valheim server. It can be any directory.
Also because I see you running on Synology; Make sure to have enough RAM (~4+ GB per Valheim server).

**Reply by MAL0W** [reply]
I could leave `/opt/valheim` as is then without changes in my current configuration, right?

I have 16GB of RAM, this is my current consumption of my Synology with 2 Valheim docker servers turned on (with CPU Intel Xeon E3-1265L-V2)
![RAM](https://user-images.githubusercontent.com/5898238/110386402-85816e80-8060-11eb-90c7-12d22d6e31e5.png)

And another question, Where do I have the server files when I don't have the `/opt/valheim` volume mounted?

I want to mount the second server to test with mods, so I would have to mount `/opt/valheim` right?

**Reply by lloesche** [reply]
Yup you can leave as is if you don't mind the downloading. 16 GB should be fine for two servers.
